cmd/bundle/deploy.go

```diff
@@ -14,20 +16,30 @@ var deployCmd = &cobra.Command{
 	RunE: func(cmd *cobra.Command, args []string) error {
 		b := bundle.Get(cmd.Context())
 
-		// If `--force` is specified, force acquisition of the deployment lock.
-		b.Config.Bundle.Lock.Force = forceDeploy
-
-		return bundle.Apply(cmd.Context(), b, bundle.Seq(
-			phases.Initialize(),
-			phases.Build(),
-			phases.Deploy(),
-		))
+		return deploy(cmd, b)
 	},
 }
 
+func deploy(cmd *cobra.Command, b *bundle.Bundle) error {
+	// If `--force` is specified, force acquisition of the deployment lock.
+	b.Config.Bundle.Lock.Force = forceDeploy
+
+	if computeID == "" {
+		computeID = os.Getenv("DATABRICKS_COMPUTE")
```

Comment:
@fjakobs @nfx  I'll change this to DATABRICKS_CLUSTER_ID for now since we have that precedent. Likely we need to change both of them over time.

Reply:
@lennartkats-db there's ClusterId on databricks config within SDK, so should be read from there. that way we get more synergy with dbconnect